recipe/meta.yaml

```diff
@@ -6,8 +6,12 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/mamba-org/mamba/archive/refs/tags/micromamba-{{ version }}.tar.gz
-  sha256: c173a3767ef7f32a7bd9f2ae1f68728fd4bc7df89b16b97c4c7f2b9fbe373898
+  - url: https://github.com/mamba-org/mamba/archive/refs/tags/micromamba-{{ version }}.tar.gz
+    sha256: c173a3767ef7f32a7bd9f2ae1f68728fd4bc7df89b16b97c4c7f2b9fbe373898
+    folder: mamba
+  # VCPKG comes with its own (short-lived) metadata which can be already outdated in the latest release
+  - url: https://github.com/microsoft/vcpkg/archive/refs/heads/master.tar.gz  # [win]
```

Comment:
```suggestion
  - url: https://github.com/microsoft/vcpkg/archive/refs/tags/2023.04.15.tar.gz  # [win]
    sha256: 7b5a7fd9a9019773e7a09dbc9aa167e01cc7745e819994d458af23e56e25e658  # [win]
```

Reply:
Oh 2023.04.15 is even not up to date -.-

Reply:
```suggestion
  - url: https://github.com/microsoft/vcpkg/archive/dc6188d0eb3f9fd7b53f4e21b1878ea868e34c02.tar.gz  # [win]
    sha256: bd68c1f05b2f38b26b447e8a9a504a0ac7c468b82237697965665f454687f9da  # [win]
```

Reply:
Unfortunately that tag is not good enough, it is the one in [vcpkg-feedstock](https://github.com/conda-forge/vcpkg-feedstock).
I believe they have pre-release tags, we could get one of them.

Reply:
Another possibility would be to release the pre-release in vcpkg-feedstock?

Reply:
> I believe they have pre-release tags

![image](https://github.com/conda-forge/micromamba-feedstock/assets/29506042/c5598878-f77f-4cef-bfc3-d51e7fdf68b3)

I don't think so.

How about we just use the latest master Git SHA at time of build like I suggested in the second suggestion and update it regularly?

Reply:
> Another possibility would be to release the pre-release in vcpkg-feedstock?

Since the end-goal is to get rid of vcpkg entirely here, I don't know if that's just one extra step that we need to take care until then...